# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/moz.build
# Commit: ccfc7c7eb397
# Full Hash: ccfc7c7eb3973902e2781ca9bbd5cfeaf2d1e264
# Author: sotaro <sotaro.ikeda.g@gmail.com>
# Date: 2021-02-23 23:03:32
# Regressor Bug: 1677459
# File Overlap Count: 1
# Description:
#   Bug 1677459 - Split platform independent part to RenderCompositorCompositorSWGL r=mattwoodrow
#   
#   The change is a preparation of Bug 1673342
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D99046
# ==============================================================================

diff -r 7a2e24377ae3 -r ccfc7c7eb397 gfx/webrender_bindings/moz.build
--- a/gfx/webrender_bindings/moz.build	Tue Feb 23 08:46:42 2021 +0000
+++ b/gfx/webrender_bindings/moz.build	Tue Feb 23 08:57:27 2021 +0000
@@ -12,6 +12,7 @@
     "RenderBufferTextureHostSWGL.h",
     "RenderCompositor.h",
     "RenderCompositorEGL.h",
+    "RenderCompositorLayersSWGL.h",
     "RenderCompositorOGL.h",
     "RenderCompositorSWGL.h",
     "RenderEGLImageTextureHost.h",
@@ -35,6 +36,7 @@
     "RenderBufferTextureHostSWGL.cpp",
     "RenderCompositor.cpp",
     "RenderCompositorEGL.cpp",
+    "RenderCompositorLayersSWGL.cpp",
     "RenderCompositorOGL.cpp",
     "RenderCompositorSWGL.cpp",
     "RenderEGLImageTextureHost.cpp",
